ZenPacks/zenoss/MySqlMonitor/tests/data/modeling_data.py

```diff
@@ -97,16 +97,16 @@
 )
 # -------------------------------------------------------------------------
 # Master results
-MASTER_STATUS1 = ({
+SOURCE_STATUS1 = ({
     'File': 'mysql-bin.000002',
     'Position': '107',
     'Binlog_Do_DB': '',
     'Binlog_Ignore_DB': '',
 },)
-MASTER_STATUS2 = ()
+SOURCE_STATUS2 = ()
 # -------------------------------------------------------------------------
 # Slave results
-SLAVE_STATUS1 = ({
+REPLICA_STATUS1 = ({
```

Comment:
Should probably have a second unit test with an updated version and replica status result.